TypeCobol.Analysis/Report/ZCallPgmReport.cs

```diff
@@ -221,123 +219,33 @@ private bool ReportCfgCallUsePoint(ControlFlowGraph<Node, DfaBasicBlockInfo<Symb
         ///     (2.1.1.2.2) The values are the values of the level 88 variable
         /// </summary>
```

Comment:
So comments above shall be moved in the ValueOrigin class

Reply:
and done